Tools/autotest/arduplane.py

```diff
@@ -1723,6 +1723,41 @@ def MainFlight(self):
         self.run_subtest("Mission test",
                          lambda: self.fly_mission("ap1.txt", strict=False))
 
+    def PitotBlockage(self):
+        '''Test detection and isolation of a blocked pitot tube'''
+        self.set_parameter("ARSPD_OPTIONS",15)
+        self.set_parameter("ARSPD_USE",1)
+        self.set_parameter("SIM_WIND_SPD",10)
+        self.set_parameter("SIM_WIND_DIR",0)
+        self.set_parameter("ARSPD_WIND_MAX",15)
```

Comment:
```suggestion
        self.set_parameters({
            "ARSPD_OPTIONS":15,
            "ARSPD_USE": 1,
            "SIM_WIND_SPD": 10,
            "SIM_WIND_DIR": 0,
            "ARSPD_WIND_MAX":15,
        })        
```

This is not a blocker, I'm just showing we have options to make things a bit more readable..